mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/okonomi/refusjon/ArrangorflateRoutes.kt

```diff
@@ -88,10 +88,11 @@ fun Route.arrangorflateRoutes() {
 
     route("/arrangorflate") {
         route("/refusjonskrav") {
-            get {
+            get("alle/{orgnr}") {
                 val arrangorIds = arrangorIderMedTilgang()
```

Comment:
Kan vi heller gjøre en `filter` eller `firstOrNull` på `arrangorIderMedTilgang` som matcher orgnr? Et orgnr mapper kun til en id uansett, det gjør databases-spørringen enklere også

Reply:
Vi kan evt. utvide `ArrangorflatePrincipal` med å inneholde id til arrangør om det også blir enklere, evt gjør et db-oppslag for å hente id, gitt orgnr, og benytte denne id'en til å hente refusjonskrav?

Reply:
Som Fredrik er inne på, så trenger vi vel ikke lenger id'ene når vi uansett endrer db-spørringen til å hente refusjonskravene gitt orgnr istedenfor.

Reply:
Funker det og. Det kan potensielt være bedre performance ved id siden dette er primary key, trenger potensielt en mindre join for å finne frem til tiktige krav (men dette avhenger av hvordan view og dataspørringer er implementert)